devops/azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+# This pipeline relies on the analysis.yml template located in the
+# repository resource descibed below
+resources:
+  repositories:
+  - repository: templates
+    type: github
+    endpoint: Team1-Project3
+    name: 1053-August-Duet-Project-Registry/project-registry-devops
+    ref: refs/heads/feature-templates
+
+# This pipeline will trigger on commits or pull-requests made to the branch-dev
+# or main branches.  Note that static analysis with SonarCloud is only performed
+# on "long-lived" branches, which are designated as 'main' and any branch name
+# prefixed with 'branch-'.
+trigger:
+- branch-dev
+- main
+pr:
+- branch-dev
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+# These variable groups are required for sonarCloud and Discord secrets to be
+# available in the analysis template
```

Comment:
you'd probably also want to document the specific variables here (so if i were just looking at this file without having the azure devops side ready, i'd know what to create).

Reply:
Updated per your suggestion